packages/main/src/Calendar.js

```diff
@@ -14,6 +14,7 @@ import DayPicker from "./DayPicker.js";
 import MonthPicker from "./MonthPicker.js";
 import YearPicker from "./YearPicker.js";
 import CalendarSelectionMode from "./types/CalendarSelectionMode.js";
+import transformDateToSecondaryType from "../../localization/src/dates/transformDateToSecondaryType.js";
```

Comment:
This is not correct, you need use the namespace @ui5/webcomponents-localization/dist

Reply:
Done